--- helpdesk.public.php	2011-12-07 14:03:24.000000000 +0100
+++ helpdesk.public.php	2011-12-07 11:01:17.000000000 +0100
@@ -90,7 +90,7 @@
 } else {
    echo "<table class='tab_cadre_central'><tr>";
    echo "<td class='top'><br>";
-   echo "<table>";
+   echo "<table style='float: left;'>";
    if (Session::haveRight('create_ticket',1)) {
       echo "<tr><td class='top' width='450px'>";
       Ticket::showCentralCount(true);
@@ -113,12 +113,7 @@
       }
       echo "</td></tr>";
    }
-
-   echo "</table></td>";
-
-   echo "<td class='top' width='450px'><br>";
-   echo "<table>";
-
+   
    // Show KB items
    if (Session::haveRight("faq","r")) {
       echo "<tr><td class='top' width='450px'>";
@@ -131,6 +126,12 @@
       echo "<tr><td>&nbsp;</td></tr>";
    }
    echo "</table>";
+   
+   $plug = new Plugin();
+   if($plug->isActivated('formcreator')) {
+      PluginFormcreatorForm::getHomeHelpdeskListForm();
+   }
+   
    echo "</td>";
    echo "</tr></table>";
 
